internal/brokers/examplebroker/broker.go

```diff
@@ -417,7 +417,7 @@ func (b *Broker) IsAuthenticated(ctx context.Context, sessionID, authenticationD
 	var authData map[string]string
 	if authenticationData != "" {
 		if err := json.Unmarshal([]byte(authenticationData), &authData); err != nil {
-			return "", "", errors.New("authentication data is not a valid json value")
+			return "", "", fmt.Errorf("authentication data is not a valid json value: %w", err)
```

Comment:
This exposes the `authenticationData` content through the JSON error, which is bad for safety. Can you switch back to the previous form?

Reply:
What I've been trying to say from the beginning of the PR review is that "a change in the proto files should not affect anything in `internal` but `internal/services`, which is where we do the communication between the service and our internal code.

This change in the error value is not triggered by your changes in the proto files, so it shouldn't be here as well.

Reply:
I'd understand if you referred to changing the behavior, but not cosmetic things.

However, let me revert this too.